### DISCUSSION TITLE
Extra puan kazanma

### DISCUSSION THREAD

**Discussion by slertolga** [IBAX-io/go-ibax#1498] [Soru Cevap - Türkçe]
**Github puan kazanmanın yolunu bilen ?**

**Comment by demircan33** [comment]
bilen varsa da söylemiyor aga

**Reply by dgv35** [reply]
yok

**Reply by bilalss18** [reply]
rez

**Reply by ibaxmustafaceviz** [reply]
ekstra puan nasıl alınır bilgisi olan yazarsa seviniriz

**Reply by MstfYtskn** [reply]
Kimse bir şey söylemek istemiyor

**Comment by Yusufknk** [comment]
İbax'ı airdrop sayfanızı githuba bağlayarak 3000 puan kazanabilirsiniz. Bunun nasıl yapılacağı ibax Türkiye telegram grubunda sabitlenmiş mesajlarda gösterilmiştir. Ekstra puanlar kazanmak istiyorsanız sistem için fikir ve önerilerinizi githubta paylaşabilirsiniz. Teknik ekip önerilerinizi dikkate aldığı takdirde puanlar da kazanabilirsiniz.

**Comment by murselbogac0** [comment]
Geleceğin projesi Ibax olur neden çünkü muhteşem yeraltiaglarina sahip
